packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -205,12 +206,12 @@ export default function Trigger(props) {
         handleKeyDown: handleKeyDownKeyboardKeys({ state, dispatch, onChangeContext }),
         handleKeyUp: handleKeyUpKeyboardKeys({ state, dispatch, onChangeContext }),
         isOpen,
-        isReadOnly,
```

Comment:
Decided not to include this – if other states like `isDisabled` and `hasError` are not included, this shouldn't be either.